src/test/java/teammates/logic/core/FeedbackResponseCommentsLogicTest.java

```diff
@@ -23,6 +33,23 @@
     private static final FeedbackQuestionsLogic fqLogic = FeedbackQuestionsLogic.inst();
     private static final FeedbackResponsesLogic frLogic = FeedbackResponsesLogic.inst();
 
+    private static final String COMMENT1_FROM_T1C1_TO_R1Q1S1C1 = "comment1FromT1C1ToR1Q1S1C1";
```

Comment:
Why all these static final variables? 

I know @daongochieu2810 mentioned to abstract out as a comment and that is a reasonable suggestion. 

The reason I don't like this is because:
1. Look at the other tests, none of them have this 
2. It is adding unnecessary clutter - there is nothing wrong with repeating the email in a test function.

Reply:
So changing them to local variable of the test function instead of having them as static variable of the class would solve the problem here?

Reply:
This is arguable but I would prefer that yes. What are your thoughts?

Reply:
Now I also think that it would be better to change it to local variables of the test function since no other test functions use them.